html/datasets.html

```diff
@@ -0,0 +1,308 @@
+    <!--<link rel="stylesheet" href="facetview2/css/facetview.css">-->
+    <!--<link rel="stylesheet" href="browsers.css">-->
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/bootstrap/css/bootstrap.min.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/bootstrap/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.min.js"></script>
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/es.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/bootstrap2.facetview.theme.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/jquery.facetview2.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/css/facetview.css">
+    <link rel="stylesheet" href="/vivo/themes/dco/css/browsers.css">
+
+
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
+            $('.facet-view-simple').facetview({
+//                search_url: '//localhost:9200/dco/dataset/_search',
+                search_url: '//dcotest.tw.rpi.edu/search/dco/dataset/_search',
+                page_size: 150,
+                sort: [
+                    {"_score" : {"order" : "desc"}}
+                ],
+                sharesave_link: true,
+                search_button: true,
+                default_freetext_fuzzify: "*",
+                default_facet_operator: "AND",
+                default_facet_order: "count",
+                default_facet_size: 15,
+                facets: [
+                    /*
+                    Facets to be used are listed below (in real order):
+                        Facet Name      |       Variable Name
+                    =======================================================
+                    Type                |   mostSpecificType
+                    Publication Year    |   publicationYear
+                    ---------------------------------------------
+                    Author              |   authors.name.exact
+                    Author Organization |   authors.organization.name.exact
+                    ---------------------------------------------
+                    DCO Community       |   dcoCommunities.name.exact
+                    Portal Group        |   portalGroups.name.exact
+                    Project             |   projects.name.exact
+                    ---------------------------------------------
+                    Data Type           |   dataTypes.name.exact
+                    =======================================================
+                    */
```

Comment:
Just to offer a more clear view of what facets are added below. Not 100% but might be useful for future users? I don't mind if we need to delete it though